pkgs/servers/nextcloud/packages/apps/recognize.nix

```diff
@@ -29,9 +34,9 @@ let
     "28" = {
       # Once this version is no longer supported, we can remove the getAppValue replacements below
       # The getAppValueString stuff will need to remain
-      version = "6.1.0";
-      appHash = "sha256-225r2JnDOoURvLmzpmHp/QL6GDx9124/YTywbxH3/rk=";
-      modelHash = "sha256-4mhQM/ajpwjqTb8jSbEIdtSRrWZEOaMZQXAwcfSRQ/M=";
+      version = "6.1.1";
```

Comment:
can't this be removed from master now that 28 is eol?

Reply:
Yes :slightly_smiling_face:  https://github.com/NixOS/nixpkgs/pull/379109/commits/12e168a1e43c7dd879f9219e4c83d8fdd49aa775